ui.js

```diff
@@ -0,0 +1,254 @@
+
+const textWindow = document.querySelector("#text-game-container");
+const textWindowText = document.querySelector("#text-screen");
+
+let closeTextWindow = () => {
+  textWindow.onclick = function () {
+    textWindow.style.display = "none";
+  };
+};
+
+closeTextWindow();
+
+let openTextWindow = () => {
+  if (game.currentState === TTT.states.P1) {
+    textWindowText.textContent = "¡Player 1 has won!  Congratulations";
+    textWindow.style.display = "flex";
+    return;
+  }
+  if (game.currentState === TTT.states.P2) {
+    textWindowText.textContent = "¡Player 2 has won!  Congratulations";
+    textWindow.style.display = "flex";
+    return;
+  }
+  if (game.currentState === TTT.states.draw) {
+    textWindowText.textContent = "¡It's been a draw!  Try again";
+    textWindow.style.display = "flex";
+    return;
+  }
+};
+
+let openTextWindow2 = () => {
+  textWindowText.textContent = "This square is already occupied";
+  textWindow.style.display = "flex";
+  return;
+};
+
+let winners = (winnerSq1, winnerSq2, winnerSq3) => {
+  if (
+    crossGame[winnerSq1].style.display === "block" &&
+    crossGame[winnerSq2].style.display === "block" &&
+    crossGame[winnerSq3].style.display === "block"
+  ) {
+    crossGame[winnerSq1].classList.add(bigCross);
+    crossGame[winnerSq2].classList.add(bigCross);
+    crossGame[winnerSq3].classList.add(bigCross);
+    boxes[winnerSq1].classList.add(WinBox);
+    boxes[winnerSq2].classList.add(WinBox);
+    boxes[winnerSq3].classList.add(WinBox);
+    playerSign[1].style.display = "block";
+    playerSign[0].style.display = "block";
+    return;
+  }
+  if (
+    circleGame[winnerSq1].style.display === "block" &&
+    circleGame[winnerSq2].style.display === "block" &&
+    circleGame[winnerSq3].style.display === "block"
+  ) {
+    circleGame[winnerSq1].classList.add(bigCircle);
+    circleGame[winnerSq2].classList.add(bigCircle);
+    circleGame[winnerSq3].classList.add(bigCircle);
+    boxes[winnerSq1].classList.add(WinBox);
+    boxes[winnerSq2].classList.add(WinBox);
+    boxes[winnerSq3].classList.add(WinBox);
+    playerSign[1].style.display = "block";
+    playerSign[0].style.display = "block";
+    return;
+  }
+};
+
+let changeToWinSymbols = () => {
+  winners(0, 1, 2);
+  winners(3, 4, 5);
+  winners(6, 7, 8);
+  winners(0, 3, 6);
+  winners(1, 4, 7);
+  winners(2, 5, 8);
+  winners(0, 4, 8);
+  winners(6, 4, 2);
+};
+
+boxes.forEach(function (box, index) {
+  box.onclick = function () {
+    const coord1 = Number(box.getAttribute("data-coord1"));
+    const coord2 = Number(box.getAttribute("data-coord2"));
+    if (game.field[coord1][coord2] !== TTT.marks.Empty) {
+      openTextWindow2();
+      return;
+    }
+    if (game.currentState == TTT.states.open) {
+      game.play(coord1, coord2);
+    }
+    if (game.field[coord1][coord2] == TTT.marks.P1) {
+      crossGame[index].style.display = "block";
+      playerSign[0].style.display = "none";
```

Comment:
I just had an idea about this and "attributes  toggle", let me explain it to you later. ***Remember me!***

Reply:
Alright